teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(age, specialization, name)
+    super(age, true, name)
+    @specialization = specialization
+  end
```

Comment:
- [ ] Please make sure to pass the default values in the initializer as requested in the project requirements. `initialize(age, specialization, parent_permission = true, name = "Unknown")`